src/main/java/graphql/schema/validation/AppliedDirectiveArgumentsAreValid.java

```diff
@@ -5,14 +5,8 @@
 import graphql.execution.NonNullableValueCoercedAsNullException;
 import graphql.execution.ValuesResolver;
 import graphql.language.Value;
-import graphql.schema.CoercingParseValueException;
-import graphql.schema.GraphQLArgument;
-import graphql.schema.GraphQLDirective;
-import graphql.schema.GraphQLInputType;
-import graphql.schema.GraphQLSchema;
-import graphql.schema.GraphQLSchemaElement;
-import graphql.schema.GraphQLTypeVisitorStub;
-import graphql.schema.InputValueWithState;
+import graphql.schema.*;
```

Comment:
Very nitpicky nitpick: instead of a wildcard import could you please import one at a time?

Reply:
very reasonable, done